source/opsmanual/keeping-software-current.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Keeping software current
+parent: /opsmanual.html
+---
+
+# Keeping software current
+
+One of our core values is to use secure and up to date software. This document lays out the recommendations for keeping our Ruby on Rails software current.
+
+## Introduction
+
+We run a lot of Rails applications. This means that we have dependencies on both Rails and Ruby versions.
+
+### Upgrading Rails
+
+It's very important that we're running a currently supported version of Rails for all applications, otherwise we aren't covered by security fixes. We should:
+
+- Be running on the current major version - this currently means 4.y.z
+- Maintain our applications at the latest current bugfix release for the minor version we're on (expressed in Gemfile syntax as: ~> X.Y.Z) - this currently means 4.1.8 and 4.2.3
```

Comment:
These version numbers are going to go out of date very quickly. Can we link to a place which lists latest releases?